api/src/main/java/org/openmrs/module/aijar/tasks/MigrateARTPatientTransfersTask.java

```diff
@@ -0,0 +1,173 @@
+package org.openmrs.module.aijar.tasks;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.hibernate.Query;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.openmrs.*;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.aijar.metadata.core.EncounterTypes;
+import org.openmrs.scheduler.SchedulerException;
+import org.openmrs.scheduler.TaskDefinition;
+import org.openmrs.scheduler.tasks.AbstractTask;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+import java.util.List;
+
+import static org.openmrs.module.aijar.AijarConstants.*;
+
+public class MigrateARTPatientTransfersTask extends AbstractTask {
+    protected final Log log = LogFactory.getLog(this.getClass());
+
+    public void execute() {
+        log.info("Start Migrate Transfer Outs From Art Summary Page To Transfer Out Encounters");
+        transferOut();
+        log.info("Completed Transfer Out Migration");
+
+        log.info("Start Migrate Transfer Ins From Art Summary Page To Transfer Out Encounters");
+        transferIn();
+        log.info("Completed Transfer In Migration");
+        stopTransferService();
```

Comment:
@slubwama why would you stop the transfer service?

Reply:
Its a migration task which gets to be stopped when its role is done